src/commands/analyze/constants.ts

```diff
@@ -0,0 +1,21 @@
+// This file contains patterns for detecting important files using a `grep` strategy
+
+// We should take into account:
+//  * common methods called to instantiate servers/conections/components such as app.listen()
+//  * common imports such as import Fastify from 'fastify' or require('fastify') (ESM and CJS)
+
+// For each file there is a list of patterns all conditions should be matched to consider the file as a candidate
+
+export const FASTIFY_PATTERN = [
+    /app\.listen|fastify\.listen/,
+    /import\s+Fastify\s+from\s+['"]fastify['"]|require\(['"]fastify['"]\)/,
```

Comment:
why capital f?

```suggestion
    /import\s+fastify\s+from\s+['"]fastify['"]|require\(['"]fastify['"]\)/,
```

Reply:
This is seems to be the object/import on the official documentation:
```
// ESM
import Fastify from 'fastify'

const fastify = Fastify({
  logger: true
})
```

https://www.npmjs.com/package/fastify